application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -33,19 +35,22 @@ public TagCommand(TagSystem tagSystem) {
 
         // TODO Thing about adding an ephemeral selection menu with pagination support
         // if the user calls this without id or similar
-        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true);
+        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true)
+            .addOption(OptionType.USER, MENTION_USER, "replying to the user ", false);
     }
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
         String id = Objects.requireNonNull(event.getOption(ID_OPTION)).getAsString();
+        User mentionUser = Objects.requireNonNull(event.getOption(MENTION_USER)).getAsUser();
         if (tagSystem.handleIsUnknownTag(id, event)) {
             return;
         }
 
         event
             .replyEmbeds(MessageUtils.generateEmbed(null, tagSystem.getTag(id).orElseThrow(),
                     event.getUser(), TagSystem.AMBIENT_COLOR))
+            .setContent(mentionUser.getAsMention())
```

Comment:
So instead, you check or `event.getOption(MENTION_USER)` is not equal to null.
If it's not equal, you run the setContent method

Otherwise, don't run the setContent method